### PR TITLE
Test index filtering Publishing API down in view

### PR DESCRIPTION
**Pull request by kevindew**
Trello: https://trello.com/c/wgkfOJsE/1283-improve-consistency-in-content-publisher-tests

The feature test that is being replaced is used to test a particular
conditional in the view. Our test strategy suggests that these should be
covered by view tests.

### CODE REVIEW

File: app/views/documents/index/_filters.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<%= form_tag({}, method: :get, data: { gtm: "filter-documents" }) do %>
+<%= form_tag(documents_path, method: :get, data: { gtm: "filter-documents" }) do %>
```
Comment (ChrisBAshton) [inline app/views/documents/index/_filters.html.erb]: Can you explain in the commit message why this change is necessary? If we're only moving/refactoring the test then we shouldn't really need to touch application code.

**Comment by kevindew** [comment]
@ChrisBAshton Thanks - just updated the commit.
